plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -7337,20 +7337,20 @@ private Answer execute(PrepareUnmanageVMInstanceCommand cmd) {
             if (targetHyperHost != null) {
                 ManagedObjectReference morTargetHostDc = targetHyperHost.getHyperHostDatacenter();
                 if (!morSourceHostDc.getValue().equalsIgnoreCase(morTargetHostDc.getValue())) {
-                    String msg = "VM " + vmName + " cannot be migrated between different datacenter";
+                    String msg = String.format("VM: %s cannot be migrated between different datacenter", vmName);
```

Comment:
`VirtualMachineTO` has a `toString` that prints id, name, uuid and type. We could use the `vmTO.toString()` here and wherever we print `vmName`.

Reply:
@GutoVeronezi `VirtualMachineTO` is not always available. It is available only in case when this method `relocateVirtualMachine` is called for `MigrateWithStorageCommand` while it remains null for `MigrateVolume` and `MigrateVmToPoolCommand`

Reply:
@shwstppr understood, thanks.